lua/powerups/loaders/sv_powerups_init.moon

```diff
@@ -15,6 +17,9 @@ include "powerups/server/grenadier.lua"
 include "powerups/server/flux_shield.lua"
 include "powerups/server/thorns.lua"
 include "powerups/server/magnetic_crossbow.lua"
+include "powerups/server/shotgun.lua"
+
+AddCSLuaFile "powerups/shared/shotgun.lua"
```

Comment:
Nit: can you put this on line 27

Reply:
I'd rather not, as it is part of a different folder, so I want to keep them more distinct.